web/packages/core/src/shadow-template.ts

```diff
@@ -336,6 +335,12 @@ ruffleShadowTemplate.innerHTML = `
 
         #local-saves {
             border-collapse: collapse;
+            overflow-y: scroll;
```

Comment:
Better to use `overflow-y: auto;` so the scrollbar is only shown when necessary, not all the time like this:
![image](https://user-images.githubusercontent.com/71368227/234397100-4be9c6d9-21d7-499b-9741-120e476a36c0.png)